pika/channel.py

```diff
@@ -225,7 +243,7 @@ def basic_cancel(self, consumer_tag='', callback=None):
         :raises ValueError:
 
         """
-        self._validate_channel()
+        self._raise_if_not_open()
```

Comment:
@lukebakken, here I renamed the method to make it explicit/self-documenting. The old name `_validate_channel` sounded more like a consistency-checker for the channel.